README.md

```diff
@@ -53,6 +53,12 @@ Displaying more detailed GitHub user data in a pinned gist.
 - [github-stats-box](https://github.com/bokub/github-stats-box) - Update a pinned gist to contain your GitHub statistics.
 - [lang-box](https://github.com/inokawa/lang-box) - Update a pinned gist to contain languages of your recent commits in GitHub
 
+## Custom
+
+Displaying data from user-defined content in a pinned gist.
```

Comment:
Hi @kf-liu - what do you think about renaming this section to be `User Defined` so it's a bit more clear?

```suggestion
## User Defined

Displaying data from user-defined (e.g. configured in the GitHub Action input parameters) content in a pinned gist.
```

Reply:
It would also be good to add this section in the `Contents` section at the top, as well.

Reply:
Both are OK for me, thanks a lot!

Reply:
cool! commit the suggestion, and I can merge in this PR for ya :)